File: interview/src/main/java/com/interview/model/Skill.java
package com.interview.model;

public class Skill {
	public String Skillname;
	private String Proficiencylevel;
	private String Positionid;
	private String registered_time;
	private String latest_updated_time;

	public Skill() {
	}

	public Skill(String Positionid, String Skillname, String Proficiencylevel) {
		this.Positionid = Positionid;
		this.Skillname = Skillname;
		this.Proficiencylevel = Proficiencylevel;
	}

	public String getPositionid() {
		return Positionid;
	}

	public void setPositionid(String positionid) {
		Positionid = positionid;
	}

	public void setSkillname(String Skillname) {
		this.Skillname = Skillname;

	}

	public String getSkillname() {
		return Skillname;
	}

	public void setProficiencylevel(String Proficiencylevel) {
		this.Proficiencylevel = Proficiencylevel;

	}

	public String getProficiencylevel() {
		return Proficiencylevel;
	}

	public String getRegistered_time() {
		return registered_time;
	}

	public void setRegistered_time(String registered_time) {
		this.registered_time = registered_time;
	}

	public String getLatest_updated_time() {
		return latest_updated_time;
	}

	public void setLatest_updated_time(String latest_updated_time) {
		this.latest_updated_time = latest_updated_time;
	}

}

File: interview_app/lib/models/Skill.dart
import 'package:flutter/material.dart';

class Skill {
  TextEditingController? skillName = new TextEditingController();
	TextEditingController? skillType = new TextEditingController();
	TextEditingController? proficiencyLevel = new TextEditingController();
	TextEditingController? positionId = new TextEditingController();
	TextEditingController? registeredTime = new TextEditingController();
	TextEditingController? latestUpdatedTime = new TextEditingController();

  Skill({
  this.skillName,
	this.skillType,
	this.proficiencyLevel,
	this.positionId,
	this.registeredTime,
	this.latestUpdatedTime,
  });

 
}
